be/src/vec/functions/function_ip.h

```diff
@@ -771,7 +771,7 @@ class FunctionIPv4CIDRToRange : public IFunction {
         ColumnWithTypeAndName& cidr_column = block.get_by_position(arguments[1]);
 
         const ColumnPtr& ip_column_ptr = ip_column.column;
-        const ColumnPtr& cidr_column_ptr = cidr_column.column;
+        const auto& [cidr_column_ptr, col_const] = unpack_if_const(cidr_column.column);
```

Comment:
do same thing to ip_column, or query like `select id, ipv4_cidr_to_range("127.0.0.1", cidr) from test_ipv4_cidr_to_range order by id` will make be crash too.